tests/queries/skip_list.json

```
@@ -106,5 +106,171 @@
+        "01305_replica_create_drop_zookeeper",
+        "01355_alter_column_with_order",
+        "01357_version_collapsing_attach_detach_zookeeper",
+        "01375_compact_parts_codecs",
+        "01378_alter_rename_with_ttl_zookeeper",
+        "01388_clear_all_columns",
+        "01396_inactive_replica_cleanup_nodes_zookeeper",
+        "01414_mutations_and_errors_zookeeper",
+        "01415_sticking_mutations",
+        "01417_freeze_partition_verbose",
+        "01417_freeze_partition_verbose_zookeeper",
+        "01430_modify_sample_by_zookeeper",
+        "01459_manual_write_to_replicas",
+        "01460_DistributedFilesToInsert",
+        "01471_calculate_ttl_during_merge",
+        "01493_alter_remove_properties_zookeeper",
+        "01355_ilike",
+        "01267_alter_default_key_columns_zookeeper",
+        "01035_concurrent_move_partition_from_table_zookeeper",
+        "01307_multiple_leaders_zookeeper",
+        "01338_long_select_and_alter_zookeeper",
+        "00534_functions_bad_arguments10",
+        "00633_materialized_view_and_too_many_parts_zookeeper",
+        "00564_versioned_collapsing_merge_tree",
+        "01033_dictionaries_lifetime",
+        "01412_cache_dictionary_race",
+        "00988_constraints_replication_zookeeper",
+        "01108_restart_replicas_rename_deadlock_zookeeper",
+        "01415_inconsistent_merge_tree_settings",
+        "00121_drop_column_zookeeper",
+        "01269_alias_type_differs",
+        "01319_manual_write_to_replicas",
+        "01280_ssd_complex_key_dictionary",
+        "00463_long_sessions_in_http_interface",
+        "01053_ssd_dictionary",
+        "01318_long_unsuccessful_mutation_zookeeper",
+        "00933_ttl_with_default",
+        "01148_zookeeper_path_macros_unfolding",
+        "01338_long_select_and_alter",
+        "01200_mutations_memory_consumption",
+        "00133_long_shard_memory_tracker_and_exception_safety",
+        "00620_optimize_on_nonleader_replica_zookeeper",
+        "01493_storage_set_persistency",
+        "01280_ttl_where_group_by",
+        "00505_shard_secure",
+        "01454_storagememory_data_race_challenge",
+        "00516_deduplication_after_drop_partition_zookeeper",
+        "01465_ttl_recompression",
+        "01085_max_distributed_connections_http",
+        "01494_storage_join_persistency",
+        "00429_long_http_bufferization",
+        "00993_system_parts_race_condition_drop_zookeeper",
+        "00305_http_and_readonly",
+        "00834_kill_mutation",
+        "00763_long_lock_buffer_alter_destination_table",
+        "01516_drop_table_stress",
+        "01457_create_as_table_function_structure",
+        "01456_modify_column_type_via_add_drop_update",
+        "01060_avro",
+        "00612_http_max_query_size",
+        "01293_system_distribution_queue",
+        "attach",
+        "ddl_dictionaries",
+        "dictionary",
+        "limit_memory",
+        "live_view",
+        "memory_leak",
```

```suggestion
        "01541_max_memory_usage_for_user",
```

P.S. Maybe it will be better to not have separate lists for this, but instead add some files, like `.reference`, i.e. `TEST.sequential` will mean that the test cannot run in parallel and so on.
That way all the information will be one place (since separate list is a separate thing that you should take care about, while the file with the same name as the test with some extension looks more cleaner)